_posts/2020-03-05-grayskull.md

```diff
@@ -0,0 +1,331 @@
+
+### Installation
+You can install ``grayskull`` using ``pip`` or ``conda``.
+
+#### With pip
+```
+pip install grayskull
+```
+
+#### With conda
+Grayskull is available on the conda-forge channel.
+```
+conda install -c conda-forge grayskull
+```
+
+
+### Grayskull vs conda-build skeleton
+
+There are some differences of recipes generated by ``grayskull`` and ``conda skeleton``.
+Taking as example the ``pytest`` recipe, which has selectors for platforms, Python versions constrains, and has several packages constrains as well.
+
+#### Grayskull (0.1.9) - took 4 seconds to generate the recipe
+
+```yaml
+{% set name = "pytest" %}
+{% set version = "5.3.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0d5fe9189a148acc3c3eb2ac8e1ac0742cb7618c084f3d228baaec0c254b318d
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - pytest=pytest:main
+    - py.test=pytest:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools >=40.0
+    - setuptools_scm
+  run:
+    - atomicwrites >=1.0      # [win]
+    - attrs >=17.4.0
+    - colorama      # [win]
+    - importlib-metadata >=0.12        # [py<38]
+    - more-itertools >=4.0.0
+    - packaging
+    - pathlib2 >=2.2.0        # [py<36]
+    - pluggy <1.0,>=0.12
+    - py >=1.5.0
+    - python
+    - wcwidth
+
+test:
+  imports:
+    - pytest
+  commands:
+    - pip check
+    - pytest --help
+    - py.test --help
+
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/pytest/
+  summary: 'pytest: simple powerful testing with Python'
+  dev_url: https://github.com/pytest-dev/pytest
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+  - marcelotrevisani
+```
+
+
+#### Skeleton (3.18.11) - took  31 seconds to generate the recipe
+
+```yaml
+{% set name = "pytest" %}
+{% set version = "5.3.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0d5fe9189a148acc3c3eb2ac8e1ac0742cb7618c084f3d228baaec0c254b318d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - atomicwrites >=1.0
+    - attrs >=17.4.0
+    - colorama;sys_platform =="win32"
+    - importlib-metadata >=0.12
+    - more-itertools >=4.0.0
+    - packaging
+    - pathlib2 >=2.2.0
+    - pip
+    - pluggy >=0.12,<1.0
+    - py >=1.5.0
+    - python
+    - wcwidth
+  run:
+    - atomicwrites >=1.0
+    - attrs >=17.4.0
+    - colorama;sys_platform =="win32"
```

Comment:
Add a note that while is this correct in the Python package metadata, this is wrong for a conda-recipe.